src/main/scala/io/iohk/ethereum/network/rlpx/RLPxConnectionHandler.scala

```diff
@@ -109,23 +110,23 @@ class RLPxConnectionHandler(nodeKey: AsymmetricCipherKeyPair)
 
     def handleTimeout: Receive = {
       case AuthHandshakeTimeout =>
-        log.debug("Auth handshake timeout")
+        log.warning(s"[Stopping Connection] Auth handshake timeout for peer $peerId")
         context.parent ! ConnectionFailed
         context stop self
     }
 
     def processHandshakeResult(result: AuthHandshakeResult, remainingData: ByteString): Unit =
       result match {
         case AuthHandshakeSuccess(secrets) =>
-          log.debug("Auth handshake succeeded")
+          log.debug(s"Auth handshake succeeded for peer $peerId")
```

Comment:
I just recalled why I haven't added peerId logging here. It's already a part of a log string - the actor name - but only in the file encoder.